<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationDialog</class>
 <widget class="QDialog" name="ConfigurationDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
          border: 2px solid rgb(218,218,218);
          border-radius: 10px;
          margin-top: 6px;
          }
          QGroupBox::title {
          subcontrol-origin: margin;
          left: 12px;
          padding: 0px 3px 0px 3px;
          }

          QScrollBar:horizontal
          {
          height: 15px;
          margin: 3px 15px 3px 15px;
          border: 1px transparent #2A2929;
          border-radius: 4px;
          background-color: #FAFAFA; /* white bg */
          }

          QScrollBar::handle:horizontal
          {
        background-color: #C2C2C2;      /* grey */
        min-width: 5px;
        border-radius: 4px;
    }

    QScrollBar::add-line:horizontal
    {
        margin: 0px 3px 0px 3px;
        border-image: url(:/UISymbols/UI_Triangle_Right.svg);
        width: 10px;
        height: 10px;
        subcontrol-position: right;
        subcontrol-origin: margin;
    }

    QScrollBar::sub-line:horizontal
    {
        margin: 0px 3px 0px 3px;
        border-image: url(:/UISymbols/UI_Triangle_Left.svg);
        height: 10px;
        width: 10px;
        subcontrol-position: left;
        subcontrol-origin: margin;
    }

    QScrollBar::add-line:horizontal:hover,QScrollBar::add-line:horizontal:on
    {
        border-image: url(:/UISymbols/UI_Triangle_Right.svg);
        height: 10px;
        width: 10px;
        subcontrol-position: right;
        subcontrol-origin: margin;
    }


    QScrollBar::sub-line:horizontal:hover, QScrollBar::sub-line:horizontal:on
    {
        border-image: url(:/UISymbols/UI_Triangle_Left.svg);
        height: 10px;
        width: 10px;
        subcontrol-position: left;
        subcontrol-origin: margin;
    }

    QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal
    {
        background: none;
    }


    QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal
    {
        background: none;
    }

    QScrollBar:vertical
    {
        background-color: #FAFAFA;  /* white */
        width: 15px;
        margin: 15px 3px 15px 3px;
        border: 1px transparent #2A2929;
        border-radius: 4px;
    }

    QScrollBar::handle:vertical
    {
        background-color: #C2C2C2;         /* grey */
        min-height: 5px;
        border-radius: 4px;
    }

    QScrollBar::sub-line:vertical
    {
        margin: 3px 0px 3px 0px;
        border-image: url(:/UISymbols/UI_Triangle_Top.svg);
        height: 10px;
        width: 10px;
        subcontrol-position: top;
        subcontrol-origin: margin;
    }

    QScrollBar::add-line:vertical
    {
        margin: 3px 0px 3px 0px;
        border-image: url(:/UISymbols/UI_Triangle_Bottom.svg);
        height: 10px;
        width: 10px;
        subcontrol-position: bottom;
        subcontrol-origin: margin;
    }

    QScrollBar::sub-line:vertical:hover,QScrollBar::sub-line:vertical:on
    {
        border-image: url(:/UISymbols/UI_Triangle_Top.svg);
        height: 10px;
        width: 10px;
        subcontrol-position: top;
        subcontrol-origin: margin;
    }

    QScrollBar::add-line:vertical:hover, QScrollBar::add-line:vertical:on
    {
        border-image: url(:/UISymbols/UI_Triangle_Bottom.svg);
        height: 10px;
        width: 10px;
        subcontrol-position: bottom;
        subcontrol-origin: margin;
    }

    QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical
    {
        background: none;
    }

    QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical
    {
        background: none;
    }</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Project Directory</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLabel" name="label_selectedPath1">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Selected path:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_projectsDir">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="scaledContents">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line_2">
             <property name="styleSheet">
              <string notr="true">    border: none;
    background: rgb(218,218,218);
    border-radius: 1px;
    max-width: 2px;
    min-width: 2px;</string>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_projectDir">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../Resources/UI_Icons.qrc">
               <normaloff>:/UISymbols/UI_Folder_Icon.svg</normaloff>:/UISymbols/UI_Folder_Icon.svg</iconset>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Classification Plugin Directory</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0,0">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QLabel" name="label_selectedPath2">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Selected path:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_classificationDir">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="scaledContents">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line_3">
             <property name="styleSheet">
              <string notr="true">    border: none;
    background: rgb(218,218,218);
    border-radius: 1px;
    max-width: 2px;
    min-width: 2px;</string>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_classificationDir">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../Resources/UI_Icons.qrc">
               <normaloff>:/UISymbols/UI_Folder_Icon.svg</normaloff>:/UISymbols/UI_Folder_Icon.svg</iconset>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Imageloader Plugin Directory</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0,0">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_13">
               <item>
                <widget class="QLabel" name="label_selectedPath3">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Selected path:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_imageLoaderDir">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="scaledContents">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line_4">
             <property name="styleSheet">
              <string notr="true">    border: none;
    background: rgb(218,218,218);
    border-radius: 1px;
    max-width: 2px;
    min-width: 2px;</string>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_imageLoaderDir">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../Resources/UI_Icons.qrc">
               <normaloff>:/UISymbols/UI_Folder_Icon.svg</normaloff>:/UISymbols/UI_Folder_Icon.svg</iconset>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Python Path</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,0">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_14">
               <item>
                <widget class="QLabel" name="label_selectedPath4">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Selected path:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_pythonPath">
                 <property name="text">
                  <string/>
                 </property>
                 <property name="scaledContents">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line_5">
             <property name="styleSheet">
              <string notr="true">    border: none;
    background: rgb(218,218,218);
    border-radius: 1px;
    max-width: 2px;
    min-width: 2px;</string>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_pythonPath">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../Resources/UI_Icons.qrc">
               <normaloff>:/UISymbols/Python_Icon.svg</normaloff>:/UISymbols/Python_Icon.svg</iconset>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">    border: none;
    background: rgb(218,218,218);
    border-radius: 1px;
    max-height: 2px;
    min-height: 2px;</string>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_error">
          <property name="styleSheet">
           <string notr="true">QLabel { color : red; }</string>
          </property>
          <property name="text">
           <string>&lt;error&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>238</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_apply">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../Resources/UI_Icons.qrc">
              <normaloff>:/UISymbols/UI_Save_Icon.svg</normaloff>:/UISymbols/UI_Save_Icon.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>19</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources/UI_Icons.qrc"/>
  <include location="../Resources/UI_Icons.qrc"/>
 </resources>
 <connections/>
</ui>
